### DISCUSSION TITLE
How to update a po translation file on WInMerge repository?

### DISCUSSION THREAD

**Discussion by slolo2000** [WinMerge/winmerge#615] [Q&A] [answered]
Hello,

I would like to update French translation but I don't know how to submit the translation.
I have read instructions here:
https://github.com/WinMerge/winmerge/blob/master/Translations/README.md

But once the french.po file translated, what is the way to submit it on WinMerge Github repository?

Thanks in advance for your help.

**Comment by sdottaka** [comment, marked as answer]
You can edit the French.po file in your browser to create a Pull Request by clicking the "Fork and edit this file" button below.

![image](https://user-images.githubusercontent.com/98126/107297602-0a578780-6ab7-11eb-9569-be2a74053881.png)
